docs/components/ModalDrawers.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: detail
+title: Navigation drawers
+description: The navigation drawer slides in from the left when the nav icon is tapped. The content should be concerned with identity and/or navigation..
+---
+
+---
+
+**Page Summary**
+
+* [Specifications references](#specifications-references)
+* [Accessibility](#accessibility)
+* [Implementation](#implementation)
+* [Component specific tokens](#component-specific-tokens)
+
+---
+
+## Specifications references
+
+- [Design System Manager - Navigation drawers](https://system.design.orange.com/0c1af118d/p/92bc26-navigation-drawers/b/146f55)
+- [Material Design - Navigation drawer](https://m2.material.io/components/navigation-drawer)
+- Technical documentation soon available
+
+## Accessibility
+
+Please follow [accessibility criteria for development](https://a11y-guidelines.orange.com/en/mobile/android/development/)
+
+## Implementation
+
+> **Jetpack Compose implementation**
+
+You can use the `OdsModalDrawer` composable like this:
+
+```kotlin
+OdsModalDrawer(
+    drawerState = rememberDrawerState(DrawerValue.Closed),// or rememberDrawerState(DrawerValue.Open)
+    content = {
+        // Do something
+    },
+    headerParametersProvider = {
+        // Do something
+    },
+    listContent = {
+        // Do something
```

Comment:
The comment `// Do something` is used for functions. Here we have composables provided by ODS Lib so you must replace theses comment by examples of usages:
- for `headerParametersProvider` (the name should be changed): You have to show an example of `OdsModalDrawerHeader` with some parameters defined cause the user of the lib should understand what to provide.
- same thing for `listContent`, provide an example of `List<OdsModalDrawerItem>`.
- the `content` should appear as a lambda at the end of the `OdsModalDrawer` call (cause you have to rearrange the parameters with `content` at the end)
```kotlin
OdsModalDrawer(
     drawerState = rememberDrawerState(DrawerValue.Closed),// or rememberDrawerState(DrawerValue.Open)
     drawerHeader = // provide an example of usage here
     listContent = // provide an example of usage here
) {
    // The content of the rest of the UI
}
```
This is done like this in Jetpack Compose SDK